roles/manage-ssh-private-keys/tasks/process-ssh-key.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Create destination path {{ ssh_key.dest | dirname }}
+  file:
+    path: "{{ ssh_key.dest | dirname }}"
+    state: directory
+    owner: "{{ ssh_key.owner }}"
+    group: "{{ ssh_key.group }}"
+    mode: 0700
```

Comment:
Maybe do the same here as what you have down below with a `default` value of `0700`. Maybe something like `ssh_key.directory_mode`